chanzuckerberg/idseq-web
# Save phylotree state

# Description

This enables saving the current metadata selection of a phylotree in the URL and in the database. 

![image](https://user-images.githubusercontent.com/28797/53130079-27879a00-351e-11e9-9114-49bc0eb9e4be.png)

## Type of change


# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. Open a phylo tree
2. Select a metadata
3. reload the page
4. see same selection
